content/introduction.html

```diff
@@ -22,23 +22,36 @@ <h1> Preface </h1>
 
 <p class="full">
 The findings in this report were derived from the results of a class survey.
-In total, 113 out of 141 (80%) graduating students responded. SE classes
-can vary significantly depending on the year, so generalize from this survey
-to other classes at your own risk.
+In total, 66 out of 92 (72%) graduating students, and 7 who were in SE but
+transferred out, responded. An addendum survey was also sent out, where
+45 more responses were retrieved from a subset of the previous respondants.
+  <br /><br />
+
+SE classes can vary significantly depending on the year, so generalize from
+this survey to other classes at your own risk.
   <br /><br />
 
 Study terms are defined as 1A, 1B, 2A, up to 4B (8 terms). 4 month co-op terms
-occur after every study term starting after 1B and ending before 4B.
+occur after every study term starting after 1B and ending before 4B. Government
+shutdowns due to the Covid-19 virus began in March 2020, partway through the 5th
+co-op term. The 4A and 4B study terms were both done remotely due to the pandemic.
+The final coop term was also conducted remotely for most students.
 <br /><br />
 
 All analysis involving income is expressed in Canadian dollars (CAD). The exchange
-rate used to convert from USD to CAD depends on time and is stated where
-relevant. All salaries include monthly housing stipend or the monthly cost of
+rate used to convert from USD to CAD depends on time so a rate of 1.3 was chosen as
+it is an approximate average of it over the past 5 years, used to make comparisons
+easier. All salaries include monthly housing stipend or the monthly cost of
```

Comment:
Is that actually true? I thought we tried for the most part to keep salary per hour and other benefits separate? So at least not applicable always. Maybe we need to come back to this later (once all the sections are finished)?

Reply:
Honestly I have no idea, the coop ones seemed to just be salary. It could explain some of the insane numbers I have been seeing if they were including their 4 month housing (I know that gets expensive quickly)